libpod/options.go

```diff
@@ -130,6 +130,19 @@ func WithConmonPath(path string) RuntimeOption {
 	}
 }
 
+// WithSyslog specifies whether the OCI Runtime should write to syslog
```

Comment:
Actually, maybe "the OCI Runtime" -> "conmon"?  Or is this making the runtime write to syslog as well?